tests/test_ops.py

```diff
@@ -40,6 +40,13 @@ def test_ops_implicit_layout():
         'maxpooling.float32',
         'resize-bilinear.float32',
         'resize-nearest-neighbor.float32',
+        'conv-reshape.float32',
+        'conv-reshape-conv.float32',
+        'conv-reshape-multiple-conv.float32',
+        # 'reshape-conv.float32',
```

Comment:
So it's blocked by our test infra which tries to convert any 4D IO from NHWC to NCHW right? Maybe we can make `Reshape` have a 2D or 3D input.

Reply:
Yeah exactly the dimension restrictions. Sorted by changing to 3D input. Thanks!